services/app-api/handlers/reports/write.test.ts

```diff
@@ -63,13 +77,26 @@ describe("Test writeReport API method", () => {
       },
       body: "{}",
     });
-    const res = await writeReport(testEvent, null);
+    const res = await writeReport(creationEvent, null);
 
     const body = JSON.parse(res.body);
     expect(res.statusCode).toBe(StatusCodes.SUCCESS);
     expect(body.status).toContain("in progress");
   });
 
+  test("Test attempted report creation with invalid data fails", async () => {
+    mockedGetReport.mockResolvedValue({
+      statusCode: 200,
+      headers: {
+        "Access-Control-Allow-Origin": "string",
+        "Access-Control-Allow-Credentials": true,
+      },
+      body: "{}",
+    });
+    const res = await writeReport(creationEventWithInvalidData, null);
+    expect(res.statusCode).toBe(StatusCodes.SERVER_ERROR);
+  });
+
```

Comment:
This is just one test of many you wrote, but great job on writing all these tests out. I'm learning so much from reading through them